coalib/bears/requirements/PythonRequirement.py

```diff
@@ -17,3 +17,66 @@ def __init__(self, package, version=""):
                         install latest.
         """
         PackageRequirement.__init__(self, 'pip', package, version)
+
+    @classmethod
+    def multiple(cls, *args):
+        """
+        Creates a tuple of multiple ``pip`` requirements, consisting of
+        ``PythonRequirements``.
```

Comment:
I think this would be easier + more precise:
`Creates a tuple of multiple ''PythonRequirements''`